### PR TITLE
BE: Profile Model

### PR DESCRIPTION
**Pull request by quddd**
Created database model for Profile (back-end)

### CODE REVIEW

**Comment by dproc96** [comment]
FYI, if you based this branch on Back-end-Signup-/-Login-API-Routes then you can make the PR directly to that branch so it's easier to see the specific changes you made.

**Review by dproc96 — APPROVED** [review]
This looks good to me, hold off on merging until Back-end-Signup-/-Login-API-Routes is merged though or merge it direct into that branch instead of dev

**Comment by quddd** [comment]
> FYI, if you based this branch on Back-end-Signup-/-Login-API-Routes then you can make the PR directly to that branch so it's easier to see the specific changes you made.

**Pull request merged**
